p4-14/v1.0.3/tex/p4.tex

```diff
@@ -2885,7 +2892,7 @@
 of \texttt{1} will match when the header is valid; \texttt{0} will match when the header is 
 not valid. As a reminder, the table does not have to explicitly include a 
 match on a field's validity to safely match on its value - invalid fields 
-will never match on a table entry that includes it. Note that metadata fields 
+will never match on a table entry that includes it (TBD: Is this true?  Or is it more accurate to say something like ``invalid fields cause the search key to contain an undefined value in those bit positions, and whether they match a table entry or not is thus unspecified''?). Note that metadata fields 
```

Comment:
@jnfoster updated this text already in his v1.0.4 branch, to delete the sentence that began "As a reminder, ...".    Looks good to me.